tests/test_metric.py

```diff
@@ -197,20 +198,25 @@ def test_msssim_analysis(self, msssim_images):
     def test_nlpd(self, einstein_img, curie_img):
         curie_img.requires_grad_()
         assert po.metric.nlpd(einstein_img, curie_img).requires_grad
+        curie_img.requires_grad_(False)
 
     def test_nspd(self, einstein_img, curie_img):
         curie_img.requires_grad_()
         assert po.metric.nspd(einstein_img, curie_img).requires_grad
+        curie_img.requires_grad_(False)
```

Comment:
why does the requires_grad need to be set to False after setting it to True first?

Reply:
It's for juggling w/ pytest fixtures. `curie_img` is used in a bunch of tests instead of instantiating new data each test.